blog/css/making-a-resizeable-element.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: Making a Resizable Element
+author: Abbas Roholamin
+category: [css]
+description: .
```

Comment:
If you add a short description about your blog, It'd be great.

Reply:
Description have also been added
you can check